File: SEM3_CODE/PHP/BASIC/10_simple_calculator.php
<?php
$firstNumber = (float) readline('Enter first number: ');    // Read the first number
$operator = readline('Enter operator (+, -, *, /): ');      // Read the operator
$secondNumber = (float) readline('Enter second number: ');  // Read the second number

if ($operator == '+') {                                     // Check for addition
    $result = $firstNumber + $secondNumber;                 // Add the numbers
} elseif ($operator == '-') {                               // Check for subtraction
    $result = $firstNumber - $secondNumber;                 // Subtract the numbers
} elseif ($operator == '*') {                               // Check for multiplication
    $result = $firstNumber * $secondNumber;                 // Multiply the numbers
} elseif ($operator == '/' && $secondNumber != 0) {         // Check for valid division
    $result = $firstNumber / $secondNumber;                 // Divide the numbers
} else {
    $result = 'Invalid operation';                          // Store an error message
}

echo 'Result: ' . $result;                                  // Display the result
?>

File: SEM5_CODE/PYTHON/BASIC/10_simple_calculator.py
first_number = float(input("Enter first number: "))    # Read the first number
operator = input("Enter operator (+, -, *, /): ")      # Read the operator
second_number = float(input("Enter second number: "))  # Read the second number

if operator == "+":                                    # Check for addition
    result = first_number + second_number              # Add the numbers
elif operator == "-":                                  # Check for subtraction
    result = first_number - second_number              # Subtract the numbers
elif operator == "*":                                  # Check for multiplication
    result = first_number * second_number              # Multiply the numbers
elif operator == "/" and second_number != 0:           # Check for valid division
    result = first_number / second_number              # Divide the numbers
else:
    result = "Invalid operation"                       # Store an error message

print("Result:", result)                               # Display the result
